src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -88,9 +88,20 @@ export class TelemetryComponent extends CdForm implements OnInit {
     this.configForm = this.formBuilder.group(controlsConfig);
   }
 
+  private replacer(key: string, value: any) {
+    if (key === 'ranges' || key === 'values') {
+      const x = [];
+      for (let i = 0; i < value.length; i++) {
+        x.push(JSON.stringify(value[i]));
```

Comment:
Is value an array or a string? 
Can you add unit tests for this function? Thank you :)!

Reply:
Ah yes that's an excellent point! The value I am hoping to change is indeed an array. I'll see what I can do to be more specific.